.circleci/config.yml

```diff
@@ -9,11 +9,13 @@ executors:
       - image: memcached
       - image: pafortin/goaws
       - image: circleci/mysql:5.7.27
+      - image: jdlk7/firestore-emulator
         environment:
           NODE_ENV: dev
           FXA_EMAIL_ENV: dev
           FXA_EMAIL_LOG_LEVEL: debug
           RUST_BACKTRACE: 1
+          AUTH_FIRESTORE_EMULATOR_HOST: localhost:9090
```

Comment:
Attaching this comment here for lack of a better place...

Where can I view the Firestore Emulator UI? I see that the emulator is running on localhost:9090. Per [these instructions](https://firebase.google.com/docs/emulator-suite/connect_and_prototype), the UI is by default at localhost:4000, but I get an “unable to connect” error in Firefox at that port.

Also when I execute an interactive shell in the docker container for the emulator, `which firebase` turns up nothing?

Reply:
Ah okay, so how can I verify the data in the Firestore emulator matches my expectations? I'm looking at the Docker image's khttps://github.com/JDLK7/firestore-emulator/blob/master/Dockerfile and at some [docs for the SDK](https://cloud.google.com/sdk/gcloud/reference/beta/emulators/firestore/) being used, and it doesn't look like there is any command than `gcloud beta emulators firestore start` which is the entry point of the docker image...